<launch>
  <node pkg="nodelet" type="nodelet" args="manager"
        name="record_player_manager" output="screen"/>
        
  <node pkg="nodelet" type="nodelet" name="cloudify"
        args="load depth_image_proc/point_cloud_xyzrgb record_player_manager --no-bond" output="screen">
    <remap from="depth_registered/image_rect" to="/gibson_ros/camera/depth/image"/>
    <remap from="depth_registered/points" to="/gibson_ros/camera/depth_registered/points"/>
    <remap from="rgb/image_rect_color" to="/gibson_ros/camera/rgb/image"/>
    <remap from="rgb/camera_info" to="/gibson_ros/camera/depth/camera_info"/>
  </node>


 </launch>